### ISSUE TITLE
Async review actions

### ISSUE THREAD

**Issue by TDaglis** [TDaglis/samplerepo#5]
(Original issue 5 created by cmohoney on 2017-09-23T17:43:27.340926+00:00)

since latency is actually a limiting factor in review speed, it would speed up the process if the page did not reload every time a video is confirmed or rejected, but rather just send an asychronous request to update the record on the server without reloading the page, so that the reviewing can immediately scroll to the next item. Once the reviewer gets to the last item and reviews it, then the page can refresh with new results

Basically it could grey out the result once a confirm or reject action has been taken. This could also be an opportunity to include an "Undo" button which would revert it back to the previous state.

Reject All however can initiate a page refresh since it is not often used

**Comment by TDaglis** [comment]
(Original issue was assigned to dipapaspyros)

(Original issue 5 last updated on 2017-09-27T12:06:19.807414+00:00)

(Issue automaticaly closed due to status in Bitbucket: resolved)

**Comment by TDaglis** [comment]
(Original comment by cmohoney on 2017-09-27T04:20:02.361741+00:00)

In testing everything seems to work fine except the Reject All action just removes the one result rather than all. I think in this case it would be best to just submit the request synchronously and have it refresh the page once the db is updated.

Once the reject all and filters are working, go ahead and roll this out!

**Comment by TDaglis** [comment]
(Original comment by dipapaspyros on 2017-09-27T12:06:19.840726+00:00)

The missing filters & reject all were implemented, this is now rolled out on production. Please test!
